tests/util-test.js

```diff
@@ -153,4 +153,23 @@ describe('util test', function () {
       expect(processDate('Sept 16, 1986')).to.equalDate(new Date('Sept 16, 1986'));
     });
   });
+
+  describe('handleInput', function () {
+    it('handles a variety of inputs', function () {
```

Comment:
can `'handles a variety of inputs'` be more informative? perhaps `"return the correct set of paths for various cli inputs"`

Reply:
I like that, updated accordingly.